src/Models/RidersModel.php

```diff
@@ -65,4 +65,96 @@ public function getAllRiders(): array|false
 
         return $allRiders;
     }
+
+    public function getTeamIdFromTeam(string $team): int|false
+    {
+        $query = $this->db->prepare('SELECT `id` FROM `teams` WHERE `team` = :team;');
+        $query->execute(['team' => $team]);
+        $data = $query->fetch();
+
+        if ($data) {
+            return $data['id'];
+        }
+        return false;
+    }
+
+    public function getNationIdFromNationality(string $nationality): int|false
+    {
+        $query = $this->db->prepare('SELECT `id` FROM `nations` WHERE `nation` = :nation;');
+        $query->execute(['nation' => $nationality]);
+        $data = $query->fetch();
+
+        if ($data) {
+            return $data['id'];
+        }
+        return false;
+    }
+
+    public function addTeam(string $team): void
+    {
+        $query = $this->db->prepare('INSERT INTO `teams` (`team`) VALUES (:team);');
+        $query->execute(['team' => $team]);
+    }
+
+    public function addNation(string $nation): void
+    {
+        $query = $this->db->prepare('INSERT INTO `nations` (`nation`) VALUES (:nation);');
+        $query->execute(['nation' => $nation]);
+    }
```

Comment:
These methods should belong to their own models, one for Team and one for Nation. Because the methods aren't doing anything to do with the riders table, they don't really belong in the model. In the future, as this project grows (imagining that it will), this RidersModel could end up growing to include a bunch of methods for other tables, so best to split it out now.